apps/roam/src/components/ResultsView/views/Kanban.tsx

```diff
@@ -279,6 +280,8 @@ const Kanban = ({
   const [isDragging, setIsDragging] = useState(false);
   const [cursorPosition, setCursorPosition] = useState({ x: 0, y: 0 });
 
+  posthog.capture("Kanban: created", { page: page });
```

Comment:
Let's nix this, it is a duplicate because "Query view: type changed" captured the layout change.